### ISSUE TITLE
Buttons in learning session

### ISSUE THREAD

**Issue by imrryr** [memphis-iis/mofacts#1512]
[statsadaptmc.zip](https://github.com/user-attachments/files/16113915/statsadaptmc.zip)
The trials do not appear as buttons, but that was how it worked 2 months ago.

Trying staging it produces a different problem... similar to what happened in MC assessment session items on production, did that fix not go to staging?
![Screenshot 2024-07-05 164743](https://github.com/memphis-iis/mofacts/assets/13023147/5f856ed7-5258-49cb-9d37-d0804609cb5f)

**Comment by MegaGeese** [comment]
This has been fixed on staging since our last deployment. If you would like then we can deploy staging.

**Comment by imrryr** [comment]
No, there is a new problem on production, see above, AND the old problem has not been fixed on staging. Did you check to confirm my report? @MegaGeese

**Comment by imrryr** [comment]
Please meet me in teams chat if you can. Both servers have different errors, both production and staging are broken in different ways, and the staging server has a the old bug @MegaGeese

**Comment by imrryr** [comment]
OK, the production bug is simple that the MC questions in a learningsession are not comping up as multiple choice. We fixed the assessmentsession MC, but this is a different context with a different problem. @MegaGeese It presents MC as if there were fill-in items.
